tasks/cephfs/test_cap_flush.py

```diff
@@ -4,9 +4,10 @@
 from textwrap import dedent
 from unittest import SkipTest
 from tasks.cephfs.fuse_mount import FuseMount
-from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from tasks.cephfs.cephfs_test_case import CephFSTestCase, long_running
```

Comment:
do you think I can change the long_running to something like {with, for, use}_teuthology as a follow up patch?